rllib/core/models/torch/heads.py

```diff
@@ -203,15 +116,6 @@ def __init__(self, config: FreeLogStdMLPHeadConfig) -> None:
         self.register_buffer("log_std_clip_param_const", self.log_std_clip_param)
 
     @override(Model)
-    def get_input_specs(self) -> Optional[Spec]:
```

Comment:
Let us check with tests, if missing a needed input or output is well traceable and the error can be well understood.